ecowitt2mqtt/hass.py

```diff
@@ -1,7 +1,7 @@
 """Define Home Assistant-related functionality."""
 import argparse
 from dataclasses import dataclass
-from typing import Any, Dict, Optional, Union
+from typing import Any, Dict, Final, Optional, Union
```

Comment:
With this change I think this now requires Python 3.8. I was originally testing with Python 3.7 and was forced to use Python 3.8 to test. So if support for Python 3.7 and below is being dropped you may want to update the README.